lib/tasks/bower-install.js

```diff
@@ -12,13 +12,20 @@ module.exports = Task.extend({
   },
   // Options: Boolean verbose
   run: function(options) {
+    var existsSync     = require('exists-sync');
```

Comment:
this can be loaded at the top of the module, as it is very fast to require and most likely already loaded.

`chalk` is lazy required, because (at-least at some point) it was extremely slow to require. If that is no-longer the case it can also be promoted to the top of the file.